python_on_whales/components/buildx/imagetools/cli_wrapper.py

```diff
@@ -52,10 +54,17 @@ def create(
             raise TypeError(
                 "The argument 'files' of the function docker.buildx.imagetools.create() must be a list of strings."
             )
+        if annotations is not None and not isinstance(annotations, dict):
```

Comment:
```suggestion
        if not isinstance(annotations, dict):
```
Following my earlier comment